tests/integration-test/src/tests/utils.rs

```diff
@@ -392,16 +408,11 @@ pub fn add_kretprobe(globals: Option<Vec<&str>>, load_type: &LoadType) -> Result
     }
 
     match load_type {
-        LoadType::Image => args.extend([
-            "--image-url",
-            KRETPROBE_IMAGE_LOC,
-            "--pull-policy",
-            "Always",
-        ]),
-        LoadType::File => args.extend(["-s", "my_kretprobe", "--path", KRETPROBE_FILE_LOC]),
+        LoadType::Image => args.extend(["--image-url", image_url, "--pull-policy", "Always"]),
+        LoadType::File => args.extend(["-s", "my_kretprobe", "--path", file_path]),
     }
 
-    args.extend(["kprobe", "-f", "try_to_wake_up", "-r"]);
+    args.extend(["kprobe", "--retprobe", "-f", "try_to_wake_up"]);
```

Comment:
I don't think this needed to change, but it should work the way you have it too.